packages/office-ui-fabric-react/src/components/Tooltip/TooltipHost.types.ts

```diff
@@ -28,12 +28,15 @@ export enum TooltipOverflowMode {
   /** Only show tooltip if parent DOM element is overflowing */
   Parent,
 
-  /** Only show tooltip if tooltip host's content is overflowing */
+  /**
+   * Only show tooltip if tooltip host's content is overflowing.
+   * Note that this does not check the children for overflow, only the TooltipHost root.
+   */
   Self
 }
 
 /**
- * Tooltip component props.
+ * TooltipHost props. All native props are passed through to the Tooltip itself.
```

Comment:
'all native props' isn't 100% clear. Is this due to the fact that it extends HTMLAttributes? Maybe there's a better, more consistent way we can make that known for all controls?

Reply:
This is a really weird case because instead of the native props going on this component (the TooltipHost), they go on its child Tooltip. I should clarify that in the comment.